mappings/net/minecraft/client/render/entity/EntityRenderDispatcher.mapping

```diff
@@ -90,6 +100,10 @@ CLASS net/minecraft/class_898 net/minecraft/client/render/entity/EntityRenderDis
 	METHOD method_3955 setRenderHitboxes (Z)V
 		ARG 1 value
 	METHOD method_3956 renderHitbox (Lnet/minecraft/class_4587;Lnet/minecraft/class_4588;Lnet/minecraft/class_1297;F)V
+		ARG 1 matrices
+		ARG 2 vertices
```

Comment:
`vertexConsumer`.

Reply:
If it's a VertexConsumer naming it `vertices` is intended.